packages/quiz/README.md

```diff
@@ -1,3 +1,55 @@
+## Overview
+
+The `quiz` package lets us add quiz questions to email sequences by linking to the questions from the ends of emails and then triggering the addition of specific tags in ConvertKit.
+
+Each of those tags can then be set up in ConvertKit to trigger the next email in the sequence. (Or anything else tags can do in ConvertKit.) This creates a better learning experience because it only sends the next email at the point someone is ready for it rather than just sending one after another regardless of whether they're ready for it.
+
+When using the quiz questions in emails, it's important to configure ConvertKit to automatically add the `ck_subscriber_id` query string value to all links in the emails. That's how the tags are tied back to the relevant subscriber.
+
+The Process for setting up quiz questions and connecting them in ConvertKit is [currently a work-in-progress and lives in Roam](https://roamresearch.com/#/app/egghead/page/GepByGFy4). It goes into_more details of how to set up the quiz questions and answers and how to wire everything up.
+
+## Quiz Configuration
+
+The top-level quiz configuration can be be found in [src/config.ts](https://github.com/skillrecordings/products/blob/main/packages/quiz/src/config.ts). It controls the product title, author, and the 'after completion messages'. The defaults are generally good, but these can be customized for individual partner products if we need to adjust the tone/voice of the messaging.
```

Comment:
this can be replaced with any configuration of the appropriate type, which might be worth noting

https://github.com/skillrecordings/products/blob/145784a99c3af849d5bd539460ba67fce1dcf02b/packages/quiz/src/config.ts#L1-L8